2020-05-31 19:21:11,225 - mid_model_train.py[line:69] - ERROR: create model error Unexpectedly found an instance of type `<class 'tuple'>`. Expected a symbolic tensor instance.
2020-05-31 19:31:28,411 - mid_model_train.py[line:132] - ERROR: training error No module named 'CCCC'
2020-05-31 19:33:24,572 - mid_model_train.py[line:132] - ERROR: training error module 'keras.losses' has no attribute 'SparseCategoricalCrossentropy()'
2020-05-31 19:39:04,168 - mid_model_train.py[line:131] - ERROR: training error 'ModelTrain' object has no attribute 'm_optimizer'
2020-05-31 19:39:31,342 - mid_model_train.py[line:131] - ERROR: training error 'ModelTrain' object has no attribute 'm_optimizer'
